pkg/apis/setup.go

```diff
@@ -95,6 +96,7 @@ func (s *Server) Setup(ctx context.Context, opts SetupOptions) (http.Handler, er
 		r.Routes(setting.Handle(opts.ModelClient))
 		r.Routes(subject.Handle(opts.ModelClient))
 		r.Routes(template.Handle(opts.ModelClient))
+		r.Routes(templateversion.Handle(opts.ModelClient))
```

Comment:
Now we've supported project scope template, may need add it to project's SubResourceHandler as well?